packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_viewport.spec.ts

```diff
@@ -118,64 +148,34 @@ context('multi-domain viewport', { experimentalSessionSupport: true }, () => {
           expect(win.innerHeight).to.equal(480)
         })
       })
-    })
 
-    it('syncs the viewport from the secondary to primary', () => {
-      const viewportChangedSpy = cy.spy()
-
-      cy.on('viewport:changed', viewportChangedSpy)
-
-      cy.switchToDomain('http://foobar.com:3500', () => {
-        // change the viewport in the secondary first
-        cy.viewport(320, 480)
-
-        cy.window().then((win) => {
-          win.location.href = 'http://localhost:3500/fixtures/multi-domain.html'
-        })
-      })
-
-      // changing the viewport to the same size shouldn't do anything
-      cy.viewport(320, 480).then(() => {
-        expect(viewportChangedSpy).not.to.be.called
-      })
-
-      cy.window().then((win) => {
-        expect(win.innerWidth).to.equal(320)
-        expect(win.innerHeight).to.equal(480)
-      })
-    })
-
-    it('syncs the viewport across multiple domains', () => {
-      cy.switchToDomain('http://foobar.com:3500', () => {
-        cy.viewport(320, 480)
-        cy.window().then((win) => {
-          expect(win.innerWidth).to.equal(320)
-          expect(win.innerHeight).to.equal(480)
+      it('syncs the viewport across multiple domains', () => {
+        cy.switchToDomain('http://foobar.com:3500', () => {
+          cy.viewport(320, 480)
+          cy.window().then((win) => {
+            expect(win.innerWidth).to.equal(320)
+            expect(win.innerHeight).to.equal(480)
+          })
         })
 
-        // navigate back to primary
         cy.window().then((win) => {
```

Comment:
I take it navigating back to the primary is no longer needed?

Reply:
nope, you can switch directly to idp